### ISSUE TITLE
show regions grouped by large region headers

### ISSUE THREAD

**Issue by cypressf** [cypressf/mit-climate-data-viz#99]
- [x] decide if it's worth copying from GCAM-LAC -> no
- [x] add headers between regions as a test
- [x] populate headers by region groups
- [x] hook up new lists to the region filter, so they hide and show regions correctly (how does GCAM-LAC do this? server.R:363)

work on branch `region-titles`, merge into `main` when done, then merge `main` into `climate-outlook`

**Comment by cypressf** [comment]
blocked waiting on official regio group colors to be in excel file
